src/lib/gql/index.ts

```diff
@@ -55,10 +55,35 @@ export function getGql(url: string, getToken: () => string) {
       }
     );
 
+  const updateCircleLogo = async (
+    circleId: number,
+    image_data_base64: string
+  ) =>
+    makeQuery(url, getToken)('mutation')(
+      {
+        upload_circle_logo: [
+          {
+            object: {
+              image_data_base64: $`image_data_base64`,
+              circle_id: $`circleId`,
```

Comment:
are the templates needed here?

Reply:
I'm on the fence about this, it seems like good practice is to always use the gql variable pattern, but its also pretty verbose. In either case, we should be consistent about it.